post-processing-service/src/main/java/gov/cdc/etldatapipeline/postprocessingservice/service/PostProcessingService.java

```diff
@@ -247,12 +248,14 @@ protected void processCachedIds() {
                     case CASE_MANAGEMENT:
                         processTopic(keyTopic, entity, ids,
                                 investigationRepository::executeStoredProcForCaseManagement);
+                        processTopic(keyTopic, entity.getName(), ids,
```

Comment:
Are we good to maintain the baseline that F_STD_PAGE_CASE updated only when there is a case_management id? That being said, is this pipeline format flow we're aiming for:
Process Investigation -> {Sends datamart topic} -> Case Management Update -> F_STD_PAGE_CASE Update -> STD/HIV Table update 🗒️

Reply:
F_STD_PAGE_CASE depends on CASE_MANAGEMENT dim which is created/updated here. Was it proper assumption to place it here?

Reply:
yes, that is how the logic seems to work.